.github/workflows/update-submodules.yml

```diff
@@ -0,0 +1,42 @@
+name: Update Library Submodules
+
+on:
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: 'Target branch for raising PR'
+        required: true
+        default: main
+      ignored:
+        description: 'Regex pattern passed to `grep -v` to ignore in the path to the library, e.g. "ota\|jobs" ignores all libraries with "ota" or "jobs" in the name.'
```

Comment:
How about having an allowlist approach of specifying the libraries to update instead of the blocklist approach? If I just have a specific library to bump the submodule pointer for, then having to provide a list of negating the other libraries can get cumbersome..

Reply:
sure